src/components/ResendVerification/ResendVerification.tsx

```diff
@@ -9,12 +10,16 @@ const ResendVerification: React.FC<VerificationEmailProps> = ({
 }: VerificationEmailProps) => (
   <>
     <div className='reset-verification'>
-      <button
-        type='button'
-        onClick={() => resendVerificationEmail(user, token)}
+      <CoolDownButton
+        label='resend'
+        duration={1000 * 120}
```

Comment:
Initially, we said 2 minutes, but after testing I feel like it might be a little long. We can reduce this to maybe 30 seconds so it's a bit nicer UI wise